pages/visualizations/viz_template.py

```diff
@@ -0,0 +1,224 @@
+from dash import html, dcc, callback
+import dash
+import dash_bootstrap_components as dbc
+from dash.dependencies import Input, Output, State
+import plotly.graph_objects as go
+import pandas as pd
+import logging
+from dateutil.relativedelta import *  # type: ignore
+import plotly.express as px
+from pages.utils.graph_utils import get_graph_time_values, color_seq
+from queries.QUERRY_USED import QUERRY_NAME as QUERRY_INITIALS
+import io
+from cache_manager.cache_manager import CacheManager as cm
+from pages.utils.job_utils import nodata_graph
+import time
+
+# TODO: REMOVE UNUSED IMPORTS
+
+page = "overview"  # EDIT FOR PAGE USED
+viz_id = "shortname-of-viz"  # UNIQUE IDENTIFIER FOR CALLBAKCS, MUST BE UNIQUE
+
+"""
+USE IN CASE OF ADDITIONAL PARAMETERS
```

Comment:
Do you have an ideas here on verbiage for extra context? I dont really have anything besides specifying that they are for callbacks. Feel like its more a dash thing than an us thing

Reply:
Oh, well maybe just something like:

"ADDITIONAL INPUT/OUTPUT PARAMETER NAMES FOR DASH"
"If you add a new Input() or Output() to your visualization callback, name them here"

that kind of thing. since we're trying to enforce the template we just need to guide them to follow that.

Reply:
@JamesKunstle got it! Added that in